src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -495,8 +499,12 @@ namespace wallet_rpc
         KV_SERIALIZE(account_index)
         KV_SERIALIZE(subaddr_indices)
         KV_SERIALIZE(priority)
+#if LOKI_DEPRECATE_AT_VERSION(7, 1, 0)
         KV_SERIALIZE_OPT(blink, false)
         KV_SERIALIZE_OPT(ring_size, (uint64_t)0)
```

Comment:
This one can also just be deleted; extra keys just get ignored in KV serialization so it shouldn't break the API to remove it even if it's being passed in.